src/EssentialSigner/index.ts

```diff
@@ -40,42 +55,70 @@ export class EssentialSigner extends Signer {
     });
   }
 
-  // Populates "from" if unspecified, and calls with the transaction
-  async call(
-    transaction: Deferrable<TransactionRequest>,
-    blockTag?: BlockTag,
-  ): Promise<string> {
-    this._checkProvider('call');
-    const tx = await resolveProperties(this.checkTransaction(transaction));
-    return await this.provider.call(tx, blockTag);
-  }
-
   // Populates all fields in a transaction, signs it and sends it to the network
   async sendTransaction(
     transaction: Deferrable<TransactionRequest>,
-  ): Promise<TransactionResponse> {
-    this._checkProvider('sendTransaction');
-    const tx = await this.populateTransaction(transaction);
-    const signedTx = await this.signTransaction(tx);
-    return await this.provider.sendTransaction(signedTx);
+  ): Promise<TransactionResponse | any> {
+    const result = await signMetaTxRequest(
+      this.provider || this.privateKey,
+      // for us, this chain ID is the chain where our Forwarding and implementation contract live.
+      // our payload is network-agnostic, so it won't necessarily be the same chain as the provider.
+      parseInt(process.env.CHAIN_ID!, 10),
+      {
+        to: transaction.to,
+        from: transaction.from,
+        ...transaction.customData, // we're using the customData override, but it might be nice to strongly type these?
+        targetChainId: process.env.CHAIN_ID,
+        data: transaction.data,
+      },
+    );
+
+    const txResult = await fetch(this.relayerUri, {
+      method: 'POST',
+      body: JSON.stringify({ ...result, forwarder: EssentialForwarder }),
+      headers: { 'Content-Type': 'application/json' },
+    })
+      .then((resp) => resp.json())
+      .then(({ result, status }) => {
+        if (status === 'success') {
+          return JSON.parse(result);
+        }
+      });
+
+    return {
+      hash: txResult.txHash,
+      confirmations: 0,
+      from: transaction.from,
+      wait: async (_confirmations?: number) =>
```

Comment:
maybe we can? not sure the internals on `wait`, should look into it.

i dont think ethers is doing anything with mempools, so perhaps wait just polls the tx for confirmations?